src/Explorer/Explorer.tsx

```diff
@@ -163,7 +163,7 @@ export default class Explorer {
   public isMongoIndexingEnabled: ko.Observable<boolean>;
   public canExceedMaximumValue: ko.Computed<boolean>;
   public isAutoscaleDefaultEnabled: ko.Observable<boolean>;
-
+  public isSchemaAnalyzerEnabled: ko.Computed<boolean>;
```

Comment:
Remove this. We are trying to reduce new dependencies on explorer. Instead check `userContext.features.enableSchemaAnalyzer` directly wherever you need it